### PR TITLE
fix #83 by rewording the -k section of the man page

### LINKED ISSUE #83: add more docs or examples for sort keys
For manual, please add more documentation or examples for sort -k sorting keys.
Especially for "data".

### PR DESCRIPTION
**Pull request by vixie**

### CODE REVIEW

File: dnsdbq.man [hunk 1]
```diff
@@ -177,9 +177,9 @@ specify newline delimited json output mode.
 .It Fl k Ar sort_keys
 when sorting with -s or -S, selects one or more comma separated sort keys,
 among "first", "last", "count", "name", and/or "data".
-If sorting was specified, but this -k option is not specified, then will default
-to sorting by first, last, count.
-Each sort key may only be specified once.
```
Comment (djw1149) [inline dnsdbq.man]: Why did you remove "Each sort key may only be specified once."?

**Comment by vixie** [comment]
On Wednesday, 18 December 2019 00:30:21 UTC djw1149 wrote:
> ...
> 
> Why did you remove "Each sort key may only be specified once."?

it's immaterial.

-- 
Paul
